Code/Framework/AzCore/AzCore/Statistics/StatisticalProfilerProxy.h

```diff
@@ -157,6 +157,22 @@ namespace AZ::Statistics
             }
         }
 
+        void GetAllStatistics(AZStd::vector<NamedRunningStatistic*>& stats)
+        {
+            for (auto& iter : m_profilers)
```

Comment:
nit: can `iter` here and below be `const auto`?

Reply:
Not without additional modifications elsewhere